proposed/simplecache.md

```diff
@@ -212,24 +212,26 @@ interface CounterInterface
     /**
      * Increment a value atomically in the cache by the given step value and return the new value
      *
-     * If the key does not exist, it is initialized to the value of $step
+     * If the key does not exist, it is initialized to the value of $step.
+     * If the value is increased above PHP_INT_MAX the return value is undefined.
      *
      * @param string $key  The cache item key
      * @param int    $step The value to increment by, defaulting to 1
      *
-     * @return int|bool The new value on success and false on failure
+     * @return int|false The new value on success and false on failure
      */
     public function increment($key, $step = 1);
 
     /**
      * Decrement a value atomically in the cache by the given step value and return the new value
      *
-     * If the key does not exist, it is initialized to the value of -$step
+     * If the key does not exist, it is initialized to the value of -$step.
+     * If the value is decreased below PHP_INT_MIN the return value is undefined.
```

Comment:
No undefined means the behavior is not defined and maybe you get PHP_INT_MAX if you get an underflow, or you get 0 or false because it got borked, or god knows what the cache backend does :)